### PR TITLE
Adding Orbit for RabbitMQ Java Client V.3.6.6

### PR DESCRIPTION
**Pull request by anupama-pathirage**

### CODE REVIEW

File: rabbit-mq/3.6.6.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.rabbitmq.wso2</groupId>
```
Comment (thusithathilina) [inline rabbit-mq/3.6.6.wso2v1/pom.xml]: group name should be org.wso2.orbit.<original group id>

File: rabbit-mq/3.6.6.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    </dependencies>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+	
+	<snapshotRepository>
```
Comment (thusithathilina) [inline rabbit-mq/3.6.6.wso2v1/pom.xml]: No need of snapshot repo

File: rabbit-mq/3.6.6.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${pom.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${pom.artifactId}</Bundle-Name>
+                        <Export-Package>
+			    com.rabbitmq.*; version="3.6.6",
+                        </Export-Package>
+                        <Import-Package>
+                            *;resolution:=optional
```
Comment (thusithathilina) [inline rabbit-mq/3.6.6.wso2v1/pom.xml]: Why do we need this?

File: rabbit-mq/3.6.6.wso2v1/pom.xml (hunk 1 above)
Comment (anupama-pathirage) [inline rabbit-mq/3.6.6.wso2v1/pom.xml]: Corrected the groupId

File: rabbit-mq/3.6.6.wso2v1/pom.xml (hunk 2 above)
Comment (anupama-pathirage) [inline rabbit-mq/3.6.6.wso2v1/pom.xml]: Removed the snapshot repo

File: rabbit-mq/3.6.6.wso2v1/pom.xml (hunk 3 above)
Comment (anupama-pathirage) [inline rabbit-mq/3.6.6.wso2v1/pom.xml]: Removed as it is not necessary for this.

**Pull request merged**